app_flutter/lib/widgets/commit_author_avatar.dart

```diff
@@ -24,21 +24,23 @@ class CommitAuthorAvatar extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     assert(commit.author.isNotEmpty);
-    final int authorHash = commit.author.hashCode;
     final String authorName = commit.author.substring(0, 1).toUpperCase();
-    Color authorColor = Color.fromRGBO(
-      authorHash & 0xFF,
-      (authorHash >> 8) & 0xFF,
-      (authorHash >> 16) & 0xFF,
-      1,
-    );
-    if (Theme.of(context).brightness == Brightness.dark) {
-      authorColor = HSLColor.fromColor(authorColor).withLightness(.75).toColor();
-    }
+    final int authorHash = authorName.hashCode;
+    final ThemeData theme = Theme.of(context);
+
+    final double hue = (360.0 * authorHash / (1 << 15)) % 360.0;
+    final double themeValue = HSVColor.fromColor(theme.backgroundColor).value;
+    Color authorColor = HSVColor.fromAHSV(1.0, hue, 0.4, themeValue).toColor();
+    if (theme.brightness == Brightness.dark) authorColor = HSLColor.fromColor(authorColor).withLightness(.65).toColor();
```

Comment:
This line seems to fail the analyzer:
```
Analyzing app_flutter...                                        

   info â€¢ Separate the control structure expression from its statement â€¢ lib/widgets/commit_author_avatar.dart:34:46 â€¢ always_put_control_body_on_new_line

1 issue found. (ran in 5.1s)
```

Reply:
Oh man I missed that one, thanks! dartfmt makes this a one-liner, but the analyzer fails it, so you have to put brackets around single lined statements to pass. 🤦 
Maybe we can unify the formatting in this repo. I spent most of my time on this change trying to navigate the eccentricities of getting the pre-submit checks to pass the formatting.